test/mcp/tools/eval_test.exs

```diff
@@ -33,22 +34,46 @@ defmodule Tidewave.MCP.Tools.EvalTest do
       Temp.add(40, 2)
       """
 
-      assert {:ok, "42", %{}} = Eval.project_eval(%{"code" => code}, Tidewave.init([]))
+      assert {:ok, json, %{}} = Eval.project_eval(%{"code" => code}, Tidewave.init([]))
+      assert Jason.decode!(json) == %{"result" => "42", "stdout" => ""}
+    end
+
+    test "suports arguments" do
+      assert {:ok, json, %{}} =
+               Eval.project_eval(
+                 %{"code" => "arguments", "arguments" => [1, "2"]},
```

Comment:
I wonder if we should restrict the arguments to strings, and if the caller wants something structured they can parse as part of eval explicitly?

Reply:
Given we go through JSON, I think anything presented in JSON is fine? Less conversion work?